ckb-bench/devtools/ci/script/benchmark.sh

```diff
@@ -6,18 +6,33 @@
 #   * AWS_SECRET_KEY, required, the AWS secret key
 #   * AWS_EC2_TYPE, optional, default is c5.xlarge, the AWS EC2 type
 #   * GITHUB_TOKEN, required, GitHub API authentication token
+#   * PGHOST, not required, postgres host to insert benchmark report to postgres in ci
+#   * PGPORT, not required, postgres host to insert benchmark report to postgres in ci
+#   * PGUSER, not required, postgres host to insert benchmark report to postgres in ci
+#   * PGPASSWORD, not required, postgres host to insert benchmark report to postgres in ci
+#   * PGDATABASE, not required, postgres host to insert benchmark report to postgres in ci
```

Comment:
The comment says that these environment variables are not required, while the script fails if they are not set, `"unbound variable"`.